src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -404,17 +404,23 @@ public int Main(string[] args)
 
             app.Command("list", c =>
             {
-                c.Description = "Print the dependencies of a given project.";
+                c.Description = "Print the dependencies of a given project";
                 var showAssemblies = c.Option("-a|--assemblies",
-                    "Show the assembly files that are depended on by given project.",
+                    "Show the assembly files that are depended on by given project",
                     CommandOptionType.NoValue);
-                var framework = c.Option("--framework",
-                    "Show dependencies for only the given framework.",
+                var framework = c.Option("--framework <NAME>",
+                    "Show dependencies for only the given framework",
                     CommandOptionType.SingleValue);
-                var runtimeFolder = c.Option("--runtime",
-                    "The folder containing all available framework assemblies.",
+                var runtimeFolder = c.Option("--runtime <PATH>",
+                    "The folder containing all available framework assemblies",
                     CommandOptionType.SingleValue);
-                var argProject = c.Argument("[project]", "The path to project. If omitted, the command will use the project in the current directory.");
+                var hideDependents = c.Option("--hide-dependents",
+                    "Hide the immediate dependents of libraries referenced in the project",
+                    CommandOptionType.NoValue);
+                var resultsFilter = c.Option("--filter <PATTERN>",
+                    "Filter the result libraries base on their names. The matching pattern supports * and ?",
```

Comment:
Result libraries doesn't look nice. Sick to `libraries referenced by the project`